superset/assets/src/dashboard/components/HeaderActionsDropdown.jsx

```diff
@@ -108,6 +109,34 @@ class HeaderActionsDropdown extends React.PureComponent {
     this.props.startPeriodicRender(refreshInterval * 1000);
   }
 
+  downloadChartImage(ev){
+    //debugger;
+    var jtarget = $(ev.currentTarget)
+                .parents('.dashboard');
+    window.setTimeout(() => {
+      // fix transparent backgrounds
+      var bgc = jtarget.css('background-color');
+      var bgcs = [bgc].concat(
+                    jtarget.parents().map((i, el) => $(el).css('background-color')).toArray()
+                  )
+                  .filter(c => c !== "rgba(0, 0, 0, 0)");    
+      var node = jtarget.css('background-color', bgcs[0]||bgc);
+      var title = jtarget.find('.dashboard-header .editable-title input').val();
```

Comment:
var => const
Also `bgc, bgcs, c`. As was mentioned in previous comment it's way better to use full word names.